tests/object/test.bats

```diff
@@ -0,0 +1,84 @@
+# PURPOSE:
+# I will test the following commands: object create, get, list, update, url, delete, and clean.
+load ../test_setup.bash
+
+# constants
+obj1="test1.txt"
+obj2="test2.txt"
```

Comment:
Nice idea to test with multiple files. Instead of hardcoding the file names - it can be easier to iterate through a directory and call the CLI commands for each file found. This means you don't have to worry about keeping the directory contents manually updated with the hardcoded file names. Create a "test_files" directory and then use a Bash loop to iterate through them in the tests? `for filename in *; do echo "put ${filename}"; done`

Reply:
Sure, I will try and implement this and let you know if I have any questions regarding this on Slack!